### ISSUE TITLE
request update MEME to 5.5.6, the current version gives inaccurate output

### ISSUE THREAD

**Issue by gy2025** [ubccr/software-layer#373]
Hi, I noticed a big difference between the output of web version of MEME (5.5.6) and the current module 5.4.1 with the exact same parameters. The software version is the only difference. Thank you, Guojun

**Comment by dsajdak** [comment]
NOTE to CCR staff - EB recipe available but will need sw version updated in it.

**Comment by gy2025** [comment]
the web version is now 5.5.7 after upgrade on 8/29

**Comment by tonykew** [comment]
MEME version 5.5.7 has been built, and is going through internal testing.
I will update this request once this version of MEME has been published.

**Comment by tonykew** [comment]
MEME version 5.5.7 has been published to the ccrsoft/2023.01 software release.

```
login1$ module spider MEME

----------------------------------------------------------------------------
  meme:
----------------------------------------------------------------------------
    Description:
      The MEME Suite allows you to: * discover motifs using MEME, DREME
      (DNA only) or GLAM2 on groups of related DNA or protein sequences, *
      search sequence databases with motifs using MAST, FIMO, MCAST or
      GLAM2SCAN, * compare a motif to all motifs in a database of motifs, *
      associate motifs with Gene Ontology terms via their putative target
      genes, and * analyse motif enrichment using SpaMo or CentriMo.

     Versions:
        meme/5.4.1
        meme/5.5.7
[...]
login1$ module spider meme/5.5.7

----------------------------------------------------------------------------
  meme: meme/5.5.7
----------------------------------------------------------------------------
    Description:
      The MEME Suite allows you to: * discover motifs using MEME, DREME
      (DNA only) or GLAM2 on groups of related DNA or protein sequences, *
      search sequence databases with motifs using MAST, FIMO, MCAST or
      GLAM2SCAN, * compare a motif to all motifs in a database of motifs, *
      associate motifs with Gene Ontology terms via their putative target
      genes, and * analyse motif enrichment using SpaMo or CentriMo.


    You will need to load all module(s) on any one of the lines below before the "meme/5.5.7" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
login1$ module load gcc/11.2.0 openmpi/4.1.1 meme/5.5.7
login1$ ls ${EBROOTMEME}/bin
ame       dust   glam2scan  mcast      momo   spamo   tomtom
centrimo  fimo   gomo       meme       purge  streme  xstreme
dreme     glam2  mast       meme-chip  sea    tgene
login1$ 
login1$ meme -version
5.5.7
login1$ 

```

**Comment by gy2025** [comment]
Hi, Thank you for installing the new version, but it's not working so far for me. Would you please keep the previous version meme/5.4.1 because my codes worked fine with the old version. I will try to figure out how to run the new version. Thank you, Guojun
________________________________
